colossalai/checkpoint_io/__init__.py

```diff
@@ -10,4 +11,5 @@
     "GeneralCheckpointIO",
     "HybridParallelCheckpointIO",
     "MoECheckpointIO",
+    "DistributedCheckpointIO",
```

Comment:
It should not be an independent checkpoint io class. It should provide some utils functions for each current checkpoint io class.